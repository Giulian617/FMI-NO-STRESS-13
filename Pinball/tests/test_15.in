100 1000
55 92 4
32 92 8
44 32 5
1 32 5
84 32 4
65 32 1
85 32 7
12 1 3
27 55 8
6 84 9
64 12 8
75 65 5
51 44 4
82 84 5
77 85 2
33 77 10
50 1 1
41 44 2
47 50 8
2 6 3
29 47 8
10 55 1
49 1 6
40 85 2
73 12 2
53 44 1
86 64 7
67 47 4
5 82 3
14 51 5
15 12 2
66 86 8
18 49 3
69 2 10
79 47 5
31 67 8
68 18 4
95 12 1
38 68 4
91 75 1
80 68 5
93 79 1
43 15 6
99 44 7
42 82 5
28 85 8
57 82 3
17 27 9
61 55 6
94 42 1
7 43 1
48 42 9
71 5 3
74 85 2
22 93 1
81 86 8
76 48 5
9 76 4
87 7 8
13 42 6
24 18 4
72 81 6
36 99 2
62 6 4
89 76 6
46 94 9
59 51 3
88 67 2
56 33 4
35 55 5
39 93 2
8 5 1
23 41 5
19 39 9
78 72 4
26 80 3
52 94 3
54 44 9
98 14 5
34 5 10
90 50 10
4 51 2
70 6 7
60 74 2
83 33 7
100 27 6
16 40 7
37 55 7
96 82 9
97 37 10
30 34 8
63 15 3
25 38 4
21 93 6
20 94 8
58 69 9
45 94 3
11 85 4
3 70 5
