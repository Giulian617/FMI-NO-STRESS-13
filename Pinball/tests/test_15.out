620444840
43482508
670421014
146242755
349051287
474906520
951279254
704690876
52581533
908556075
920587049
338503884
657552438
921136295
955886190
554296191
32651159
574133727
573525323
908181835
574783954
650489630
282613869
99272123
227651173
99272123
157382662
546431639
146249018
927589896
582303231
991986824
704690876
326755144
75030807
36563406
943752732
402387432
166718709
943752732
691362103
49242797
99117430
84567673
387173280
575592609
453370655
671966959
951002670
345745587
447069680
